searchlib/src/vespa/searchlib/diskindex/fieldreader.cpp

```diff
@@ -124,11 +126,9 @@ FieldReader::open(const vespalib::string &prefix,
                   const TuneFileSeqRead &tuneFileRead)
 {
     vespalib::string name = prefix + "posocc.dat.compressed";
-    FastOS_StatInfo statInfo;
 
-    bool statres = FastOS_File::Stat(name.c_str(), &statInfo);
-    if (!statres) {
-        LOG(error, "Could not stat compressed posocc file %s: %s", name.c_str(), getLastErrorString().c_str());
+    if (!fs::exists(fs::path(name))) {
+        LOG(error, "Compressed posocc file %s does not exist: %s", name.c_str(), getLastErrorString().c_str());
```

Comment:
Will the output of `getLastErrorString()` (i.e. `errno`) make sense here, or should we just skip it? Since this is the non-`std::error_code` version of `exists()` it will throw on OS errors.

Reply:
Fixed